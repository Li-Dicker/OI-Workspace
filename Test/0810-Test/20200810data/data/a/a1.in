10
9
89264 24051 87471 6085 71366 84791 83564 87600 40460 
19011 26139 40153 16343 31719 11256 10658 14294 3769 
10
30046 72753 82207 23377 67745 49269 67504 17421 69986 40538 
46195 28248 43341 16831 48920 20181 35191 17493 20208 6181 
10
70973 25776 16176 69549 6381 69052 23126 86757 23837 81805 
23959 44065 29434 49556 26963 39495 24478 15028 36981 40177 
9
31771 5497 97764 73685 46537 92770 85599 21113 26582 
41463 31026 3663 36946 20596 12238 15512 23175 10520 
9
89906 25650 58665 76444 65923 97790 12756 86900 29339 
11098 16021 34726 16730 8459 16266 38870 1792 42553 
9
45570 29489 26313 97774 69912 576 76965 6066 38570 
29424 20392 10678 4778 7959 15370 40490 16723 32030 
9
29966 96390 4428 26495 12433 6885 33396 40450 86897 
9378 42520 20055 24181 4859 49901 881 18157 22199 
10
94194 37841 60367 4611 26244 72638 18852 59177 83869 40828 
23693 21503 19445 34686 21660 2221 43193 16001 12198 49147 
10
69704 12846 3197 76753 68215 43768 71217 47716 4531 34308 
14076 40169 6767 18324 36635 49537 40496 36820 8211 7827 
9
47056 25493 78148 89208 83532 19548 43462 74153 50523 
559 11422 41394 1564 17489 28368 47769 44335 27890 
