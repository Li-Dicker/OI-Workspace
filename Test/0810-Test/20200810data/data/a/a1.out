Case #1: 724863
Case #2: 745532
Case #3: 754460
Case #4: 610755
Case #5: 649227
Case #6: 504006
Case #7: 436442
Case #8: 645196
Case #9: 656655
Case #10: 666316
